sklearn/model_selection/_split.py

```diff
@@ -419,6 +420,71 @@ def _iter_test_indices(self, X, y=None, groups=None):
             current = stop
 
 
+class RepeatedKFold(_BaseKFold):
+    """ Repeated KFold cross-validation
+
+    To repeat KFold n times with different split at each repetition
+    (by default).
+
+    Read more in the :ref:`User Guide <cross_validation>`.
+
+    Parameters
+    ----------
+    n_reps : int, default=2
+        Number of times KFold needs to be repeated.
+
+    n_splits : int, default=3
```

Comment:
It would be a bit inconsistent, but if I could go back, I'd rather have this as 5 in KFold. Do we maybe want 5 here?